### DISCUSSION TITLE
Invoke-PnpBatch Details doesn't seem to return any details

### DISCUSSION THREAD

**Discussion by teamneroblanco** [pnp/powershell#1553] [Ideas]
When I run a batch with the -Details parameter I don't see any details returned at all?

V1.9.0

**Comment by JimBobUKII** [comment]
Same here, version 1.9.0

**Comment by ricbrady** [comment]
It works for me with Version 1.11.0 if I do it so (see below).
My Problem is that I have no one 'BatchRequest' in the returned List. Is this a Bug?

$details = Invoke-PnPBatch -Batch $batch -Details
foreach ($detail in $details) {
            If ($detail -is [PnP.PowerShell.Commands.Base.BatchResult]) {
                write-host "Batch exception: ERROR: $($detail.ErrorMessage) - RESPONSECODE: $($detail.ResponseCode) - REQUEST: $($detail.Request)"
            }
            If ($detail -is [System.Collections.Generic.List[PnP.PowerShell.Commands.Model.BatchRequest]]) {
                foreach ($request in $detail) {
                    write-host "Processing CSV File: HttpStatusCode: $($request.HttpStatusCode) - ResponseJson: $($request.ResponseJson)"
                }
            }
        }

**Comment by broilertech** [comment]
I dont't see any details... Is it possible to get the IDs of new created Items after batch creation?

Add-PnPListItem -List $list -Values $values1 -Batch $batch
Add-PnPListItem -List $list -Values $values2 -Batch $batch
$details Invoke-PnPBatch -Batch $batchRisk -Details
foreach ($detail in $details) { $detail ????  }

**Reply by AndersRask** [reply]
No, since this is done asynchronously I don' think you can?!
